### ISSUE TITLE
Initial data commit on `data_axondeepseg_stanford`

### ISSUE THREAD

**Issue by hermancollin** [neuropoly/data-management#277]
Hi! 
The branch is `ac/init`. It contains all the BIDS files, images and json sidecars. Annotations will eventually be added for some samples.
The dataset should pass the bids validator.

**Comment by mguaypaq** [comment]
Since the subjects don't all have the same set of samples, I added the following `.bids-validator-config.json`:
```json
{"ignore":["INCONSISTENT_SUBJECTS"]}
```
(Most subjects have 8 samples. But `sub-366A` has 16 samples, and `sub-368A` has 4 samples.)

I also marked your local clone of the repository as "dead", so that other people don't try to get data files (unsuccessfully) from your laptop when they run `git annex get`.

After that, both git-annex and bids-validator are fine, so I merged into `master` and deleted the branch `ac/init`.

It's cool that this dataset is MIT-licensed!

**Comment by hermancollin** [comment]
@mguaypaq Thanks for sharing the bids-validator-config trick. I didn't about this.
> I also marked your local clone of the repository as "dead"

oops sorry I thought I did `git annex dead here`.

About the MIT license, I actually didn't mean to put that there :sweat_smile: I think I copied files from another BIDS dataset and added it in a commit by mistake. Although I'd like this to be MIT-licensed, we should discuss this with our collaborator first to see if it's even possible. I guess it's not a big issue for now since the data is privately hosted.
